python/ray/data/dataset.py

```diff
@@ -897,6 +898,150 @@ def add_column(batch: DataBatch) -> DataBatch:
             **ray_remote_args,
         )
 
+    @PublicAPI(api_group=BT_API_GROUP)
+    def distinct(
+        self, subset: Optional[List[str]] = None, keep: Union[str, bool] = "first"
```

Comment:
```suggestion
        self, keys: Optional[List[str]] = None, keep: Union[Literal["first", "last"], bool] = "first"
```
`keys` would be more clear and more consistent with other APIs

Reply:
Good suggestion, updated.